libpod/reset.go

```diff
@@ -122,9 +123,23 @@ func (r *Runtime) Reset(ctx context.Context) error {
 		}
 	}
 	if storageConfPath, err := storage.DefaultConfigFile(rootless.IsRootless()); err == nil {
-		if _, err = os.Stat(storageConfPath); err == nil {
+		if _, err := os.Stat(storageConfPath); err == nil {
 			fmt.Printf("A storage.conf file exists at %s\n", storageConfPath)
-			fmt.Println("You should remove this file if you did not modify the configuration.")
+			fmt.Println("you can remove it or override it with --run-root and --graph-root")
```

Comment:
I don't like this change, I want users to remove the storage.conf file, and not use --run-root or graphroot.
Also if I am doing updateConf, why print these messages at all.

Reply:
Ok, I can revert this message